spec/index.bs

```diff
@@ -1131,14 +1163,30 @@ An [=adapter=] has the following internal slots:
 
 [=Adapters=] are exposed via {{GPUAdapter}}.
 
+<div algorithm data-timeline=device>
+    A given {{GPUAdapter}} |adapter| is <dfn abstract-op>expired</dfn> if all of the requirements of
+    the following [=device timeline=] steps are met:
+
+    <div class=validusage>
+        - |adapter|.{{GPUAdapter/[[adapter]]}}.{{adapter/[[expired]]}} must be `true`
+    </div>
```

Comment:
I don't think validusage makes sense here since expired=true is the _invalid_ usage.

I think just flatten this down, and then it's a non-issue, unless you want to keep the `[=device timeline=] steps` part.
```suggestion
    A given {{GPUAdapter}} |adapter| is <dfn abstract-op>expired</dfn> if
    |adapter|.{{GPUAdapter/[[adapter]]}}.{{adapter/[[expired]]}} is `true`.
```